teamA/jeongHaeMan1/app.js

```diff
@@ -0,0 +1,50 @@
+require('dotenv').config();
+
+const http = require('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized")
+    })
+    .catch((err) => {
+        console.error("Error during Data Source initializtion", err)
+        myDataSource.destroy()
+    })
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(''));
```

Comment:
morgan 미들웨어의 format을 지정하지 않는다면, default format이 적용됩니다.
서버를 실행하여 터미널 상 로그를 확인해보세요!  morgan 미들웨어 관련된 로그가 한 줄 나올 것입니다.

morgan format에 대해 찾아보고 큰따옴표를 제거하거나, 다른 format을 지정해주세요~!

아래는 예시입니다!

```suggestion
app.use(morgan('dev'));
```